tests/validation_tests/test_pre_checks.py

```diff
@@ -279,3 +279,28 @@ def test_full_pre_transformation_validation_pipeline_failure_place_vs_fund(valid
             expected_values={"TD"},
         )
     ]
+
+
+def test_full_pre_transformation_validation_pipeline_failure_place_null(valid_workbook_round_four):
+    valid_workbook_round_four["2 - Project Admin"][4] = [
+        "",
+        "",
+        "",
+        "",
+        "",
+        "Future_High_Street_Fund",
+        "",
+        "",
+        "",
+    ]
```

Comment:
Should we add a test for if both are null?

Reply:
I don't think it makes a difference to this specific bug as if fund_type is None the failure won't arise; however, if place_name is None (or not valid), then the look-up of PLACE_NAME_TO_FUND_TYPE will result in None, and whilst ```is None in []" won't raise an error, ```is "TD" in None``` will

However, I don't think it will hurt, and good to have the coverage whilst making this change anyway. Will add